docs/user_guide/building_blocks/categories.md

```diff
@@ -1,3 +1,525 @@
+Work in Progess! This guide has not been finalized and the presented code has not yet been tested.
+
+TODO: make the code testable
+
 # Categories
 
-TODO
+In columnflow, there are many tools to create a complex and flexible categorization of all analysed
+events.
+Generally, this categorization can be layered.
+We refer to the smallest building block of these layers as **leaf categories**, which can subsequently
+be either run individually or combined into more complex categories.
+This guide presents how to implement a set of categories in columnflow and shows how to use the
+resulting categories via the {py:class}`~columnflow.tasks.yields.CreateYieldTable` task.
+
+
+
+## Create a category
+
+If you used the analysis template to setup your analysis, there are already two categories
+included, named ```incl``` and ```2j```.
+To test that the categories are properly implemented,
+we can use the CreateYieldTable task:
+```shell
+law run cf.CreateYieldTable --version v1 \
+    --calibrators example --selector example --producers category_ids \
```

Comment:
Shouldn't this also be 

```suggestion
    --calibrators example --selector example --producers example \
```

?

Alternatively, we could also first call `ProduceColumns` if you want to highlight the `category_ids` producer, or simply mention it in the text (which you do later on in any case)